CMakeLists.txt

```diff
@@ -47,6 +47,7 @@ target_link_libraries(${PROJECT_NAME}
   pcsc-cpp
   ${CMAKE_DL_LIBS}
   OpenSSL::Crypto
+  $<$<C_COMPILER_ID:MSVC>:Shlwapi>
```

Comment:
Wouldn't it be cleaner to add the linking directive using `#pragma comment(lib, "Shlwapi")` directly in `src/electronic-ids/pkcs11/Pkcs11ElectronicID.cpp` instead in `CMakeLists.txt`?

See example [here](https://github.com/web-eid/libelectronic-id/pull/2/files#diff-1d1e9d13402083e3e4cfef76240e0afa5c7dbd2c004dbd19153acbe3703be28dR32).